server/server.go

```diff
@@ -53,38 +59,27 @@ func (cs *cribbageServer) Serve() {
 }
 
 func (cs *cribbageServer) ginPostCreateGame(c *gin.Context) {
-	var pIDs []model.PlayerID
-
-	pID := getPlayerID(c, `player1`)
-	if pID == model.InvalidPlayerID {
-		c.String(http.StatusBadRequest, `Needs player1`)
-		return
-	}
-	pIDs = append(pIDs, pID)
-
-	pID = getPlayerID(c, `player2`)
-	if pID == model.InvalidPlayerID {
-		c.String(http.StatusBadRequest, `Needs player2`)
+	var gameReq model.Game
+	// jsonutils.GameBinding is a custom binding interface to handle the game actions appropriately
+	err := c.BindWith(&gameReq, jsonutils.GameBinding)
```

Comment:
lol and now we shouldn't need a custom bind with cuz the "create game request" should only have player IDs on it:#

Reply:
🤦‍♂️ I'm embarrassed to admit how much time I spent testing that